server/main.lua

```diff
@@ -1,11 +1,4 @@
 ESX                = nil
-PlayersHarvesting  = {}
```

Comment:
Removing this function will disable another script such as esx_mechtoolkit

Reply:
Yes po. esx_mechtoolkit won't be used anymore. Crafting and use of repairkit and bodykit will be disabled as it may be abused by non-mechanics.

Reply:
instead of deleting the code on the server-side which is critical in the integrity of the script. You can disable the function on the client-side.

Reply:
Reverted to original file, disabled function on client side.